packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -450,10 +450,13 @@ class _CupertinoBackGestureDetectorState extends State<_CupertinoBackGestureDete
       fit: StackFit.passthrough,
       children: <Widget>[
         widget.child,
-        new Listener(
-          onPointerDown: _handlePointerDown,
-          behavior: HitTestBehavior.translucent,
-          child: new SizedBox(width: _kBackGestureWidth),
+        new Align(
```

Comment:
In fact we can use a Positioned with an explicit `width` and get rid of the `Container` entirely.